ddtrace/ext/git.py

```diff
@@ -143,7 +143,11 @@ def extract_user_info(cwd=None):
 
 def extract_git_version(cwd=None):
     output = _git_subprocess_cmd("--version")
-    version_info = tuple([int(part) for part in output.split()[-1].split(".")])
+    try:
+        version_info = tuple([int(part) for part in output.split()[2].split(".")])
+    except ValueError:
+        log.error("Git version not found, it is not following the desired version format.")
```

Comment:
nit: This will help us debug issues in the future
```suggestion
        log.error("Git version not found, it is not following the desired version format: {}".format(output))
```

Reply:
Can we add a regression test where `_git_subprocess_cmd()` returns an invalid string. We don't want this to fail downstream.  We should still generate valid spans if the git version returns `0.0.0`